Initial Commit

ssmusoke opened the pull request:
https://trello.com/c/8irGbuH8

ssmusoke committed:
Initial Commit

ssmusoke committed:
Initial Commit

ssmusoke committed:
Module Setup and Configuration
Addition of overridden forms and concept metadata

ssmusoke committed:
Module Setup and Configuration
Addition of overridden forms and concept metadata

ssmusoke committed:
parent 9d1fdbe9d576ea41e33da9c69290f4196637b959
author Stephen S. Musoke <ssmusoke@gmail.com> 1548243359 +0300
committer Stephen S. Musoke <ssmusoke@gmail.com> 1550653177 +0300

Module Setup and Configuration
Addition of overridden forms and concept metadata

ssmusoke committed:
Merge branch 'master' of https://github.com/METS-Programme/openmrs-module-mulagoemr

# Conflicts:
#	README.md

slubwama commented on api/src/main/java/org/openmrs/module/mulagoemr/MulagoEMRActivator.java:
```diff
@@ -0,0 +1,37 @@
+	 * @see #shutdown()
+	 */
+	public void shutdown() {
+		log.info("Shutdown Uganda Prisons Services Electronic Medical Records System ");
```
Change to Mulago

slubwama commented on api/src/main/java/org/openmrs/module/mulagoemr/MulagoEMRActivator.java:
```diff
@@ -0,0 +1,37 @@
+	 * @see #shutdown()
+	 */
+	public void shutdown() {
+		log.info("Shutdown Uganda Prisons Services Electronic Medical Records System ");
```
Change to Mulago

ssmusoke replied:
@slubwama Done

slubwama commented on api/src/main/java/org/openmrs/module/mulagoemr/MulagoEMRSystemConfig.java:
```diff
@@ -0,0 +1,21 @@
+@Component("mulagoemr.UgandaPrisonsServicesElectronicMedicalRecordsSystemConfig")
+public class MulagoEMRSystemConfig {
+	
+	public final static String MODULE_PRIVILEGE = "Uganda Prisons Services Electronic Medical Records System  Privilege";
```
Change to Mulago

ssmusoke replied:
@slubwama Done

slubwama commented on api/src/main/resources/messages.properties:
```diff
@@ -0,0 +1,2 @@
+mulagoemr.title=Uganda Prisons Services Electronic Medical Records System 
```
Change to Mulago

ssmusoke replied:
@slubwama Done

slubwama commented on api/src/main/resources/messages_es.properties:
```diff
@@ -0,0 +1 @@
+mulagoemr.title=Uganda Prisons Services Electronic Medical Records System 
```
Change to Mulago

ssmusoke replied:
@slubwama Done

slubwama commented on api/src/main/resources/messages_fr.properties:
```diff
@@ -0,0 +1 @@
+mulagoemr.title=Uganda Prisons Services Electronic Medical Records System 
```
Change to Mulago

ssmusoke replied:
@slubwama Done

slubwama reviewed (changes requested):
https://pastebin.com/p9UPSsnz
Consider the concept UUIDs as well

ssmusoke committed:
Removed all references to source module from which changes were copied

ssmusoke committed:
Removed all references to source module from which changes were copied

ssmusoke committed:
Updated the custom concepts that had CIEL UUIDs causing a unique index error

ssmusoke committed:
Updated the custom concepts that had CIEL UUIDs causing a unique index error

ssmusoke commented:
@slubwama good catch on the concept UUIDS I have updated the SQL script to change those with CIEL style uuids

slubwama commented:
@ssmusoke  keep it one commit

ssmusoke committed:
Add the custom concepts for MUJHU

ssmusoke committed:
Add the custom concepts for MUJHU

slubwama commented on api/src/main/java/org/openmrs/module/mulagoemr/MulagoEMRActivator.java:
```diff
@@ -0,0 +1,46 @@
+		// install concepts
+		DataImporter dataImporter = Context.getRegisteredComponent("dataImporter", DataImporter.class);
+		
+		dataImporter.importData("MulagoEMRConcepts.xml");
```
Change this to the Correct Path. This is pointing to a different location.
https://pastebin.com/zyyC3N4y

ssmusoke replied:
@slubwama I don't know how I missed that, thanks for catching this. I have also updated the pom files to align to aijar so that it is consistent for later use.

ssmusoke committed:
Correct the path to custom concepts in the Activator, and add error handling
Remove CIEL concept uuids for the concept names
Add packaging of the resources just like in AIJAR

ssmusoke committed:
Correct the path to custom concepts in the Activator, and add error handling
Remove CIEL concept uuids for the concept names
Add packaging of the resources just like in AIJAR

slubwama commented:
@ssmusoke Kindly squash your commits into one

ssmusoke committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	README.md

ssmusoke committed:
Delete IDEA project files
